samples/bluetooth/bap_broadcast_assistant/src/main.c

```diff
@@ -568,7 +576,6 @@ BT_CONN_CB_DEFINE(conn_callbacks) = {
 int main(void)
 {
 	int err;
-	struct bt_bap_broadcast_assistant_add_src_param param = { 0 };
```

Comment:
Why was this moved from here to the global scope? Seems like it would be reset in either case

Reply:
It was moved to be consistent with the other data areas.
Where is it reset when it is only in the main?

Reply:
You are correct. It should probably have been moved to the loop so that it's 0-initialized for each iteration